python/cudf/cudf/tests/test_parquet.py

```diff
@@ -4373,6 +4374,10 @@ def test_parquet_reader_mismatched_nullability_structs(tmpdir):
     )
 
 
+@pytest.mark.skipif(
+    platform.machine() == "aarch64",
```

Comment:
Can we also skip this only if the version is 19.0.0? We should make sure this fix gets into later releases.

Reply:
yup